pennylane/resource/specs.py

```diff
@@ -193,14 +144,12 @@ def specs(
         qnode (.QNode | .QJIT): the QNode to calculate the specifications for.
 
     Keyword Args:
-        level (str, int, slice): An indication of what transforms to apply before computing the resource information.
-            Check :func:`~.workflow.get_transform_program` for more information on the allowed values and usage details of
-            this argument.
-        compute_depth (bool): Whether to compute the depth of the circuit. If ``False``, the depth will not be included in the returned information.
+        level (str | int | slice | iter[int]): An indication of what transforms to apply before computing the resource information.
+        compute_depth (bool): Whether to compute the depth of the circuit. If ``False``, the depth will not be included in the returned information. Default: True
 
     Returns:
         A function that has the same argument signature as ``qnode``. This function
-        returns a dictionary (or a list of dictionaries) of information about qnode structure.
+        returns a :class:`~.resource.CircuitSpecs` object containing information about qnode structure.
```

Comment:
Perhaps also needs to be updated to say that it could return a list of `CircuitSpecs` in certain cases.
```suggestion
        returns a :class:`~.resource.CircuitSpecs` object containing the ``qnode`` specifications.
```